src/main/java/org/opentripplanner/standalone/server/OTPApplication.java

```diff
@@ -92,7 +92,7 @@ public OTPApplication (OTPServer server) {
 
         if (OTPFeature.ActuatorAPI.isOn()) {
             singletons.add(new MetricsApplicationEventListener(
-                    ActuatorAPI.prometheusRegistry,
+                    Metrics.globalRegistry,
```

Comment:
Does adding the metrics application listener need to be dependent on `OTPFeature.ActuatorAPI.isOn()`?

Can it always be enabled, similar to the `new MetricsLogging(server);`.

Reply:
Good point, fixed.